corehq/apps/cloudcare/static/cloudcare/js/formplayer/router.js

```diff
@@ -84,43 +84,28 @@ hqDefine("cloudcare/js/formplayer/router", function () {
          * be a form response which will route to a new form.
          */
         renderResponse: function (response) {
-            var currentFragment,
-                urlObject,
-                encodedUrl,
-                menuCollection;
-
-            currentFragment = Backbone.history.getFragment();
-            urlObject = Util.CloudcareUrl.fromJson(Util.encodedUrlToObject(currentFragment));
-            if (urlObject.appId) {
-                // will be undefined on urlObject when coming from an incomplete form
-                response.appId = urlObject.appId;
-            }
-
              if (response.notification) {
                 FormplayerFrontend.trigger("handleNotification", response.notification);
              }
 
             // When the response gets parsed, it will automatically trigger form
             // entry if it is a form response.
-            menuCollection = hqImport("cloudcare/js/formplayer/menus/collections")(
+            let menuCollection = hqImport("cloudcare/js/formplayer/menus/collections")(
                 response,
                 { parse: true }
             );
             // Need to get URL fragment again since fetch might have updated it
-            currentFragment = Backbone.history.getFragment();
-            urlObject = Util.CloudcareUrl.fromJson(Util.encodedUrlToObject(currentFragment));
-            encodedUrl = Util.objectToEncodedUrl(urlObject.toJson());
-            FormplayerFrontend.navigate(encodedUrl);
+            Util.setUrlToObject(Util.currentUrlToObject());
```

Comment:
this seems like a no-op to me

Reply:
I don't get the purpose of this, either, but since it came from https://github.com/dimagi/commcare-hq/pull/18921 it'd be good to have QA run some EOF nav tests.